Fix inclusive language issues (#2790)
theforeman/foreman-documentation: guides/common/modules/proc_configuring-ipxe-to-reduce-provisioning-times.adoc

You can use {Project} to configure PXELinux to chainboot iPXE in BIOS mode and boot using the HTTP protocol if you have the following restrictions that prevent you from using PXE:

* A network with unmanaged DHCP servers.

> * A PXE service that is blocklisted on your network or restricted by a firewall.

Reviewer: I dislike the allowlist/denylist terminology. Can't we be more generic here and simply say that is somehow unreachable? Because that's the core of the issue.
```suggestion
* An unreachable PXE service, for example by a firewall restriction.
```

Author: Works for me.
